calm-business/sass/typography/_headings.scss

```diff
@@ -16,7 +16,6 @@
 .pagination .nav-links,
 .sticky-post,
 .site-title,
-.site-title:focus,
```

Comment:
the compiled css does not have these changes: https://opengrok.a8c.com/source/xref/pub/calm-business/style.css#455

Reply:
I'm actually finding all of these changes from _headings.scss in the style.css file: https://opengrok.a8c.com/source/xref/pub/calm-business/style.css#564
Going to remove this change and instead push these scss changes to SVN.